scripts/sct_dmri_moco.py

```diff
@@ -307,9 +307,8 @@ def main(args=None):
     elapsed_time = time.time() - start_time
     sct.printv('\nFinished! Elapsed time: ' + str(int(round(elapsed_time))) + 's', param.verbose)
 
-    # To view results
-    sct.printv('\nTo view results, type:', param.verbose)
-    sct.printv('fslview -m ortho,ortho ' + param.path_out + file_data + param.suffix + ' ' + file_data + ' &\n', param.verbose, 'info')
+    # display viewer syntax
+    sct.display_viewer([param.path_out + file_data + param.suffix, file_data], mode='ortho,ortho')
```

Comment:
Suggest again to create a local variable with a good name instead of passing a constructed filename here.

Reply:
ok

Reply:
fixed in 47d16609b5e5e364c294b266b31c600e9dcdd8d6